### ISSUE TITLE
py.path.move() across disks fails on Windows in Python 3.4

### ISSUE THREAD

**Issue by pytestbot** [pytest-dev/py#80]
- Bitbucket: https://bitbucket.org/pytest-dev/py/issue/80
- Originally reported by: @smsearcy
- Originally created at: 2016-04-29T01:39:09.422

Attempting to `move()` a file via `py.path` in Windows across disks fails in Python 3.4.3 (it works in Python 2.7.11).

```
(venv34) PS E:\src\python\py> python
Python 3.4.3 (v3.4.3:9b73f1c3e601, Feb 24 2015, 22:43:06) [MSC v.1600 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import py.path
>>> pth = py.path.local(r'e:\tmp\testing.txt')
>>> pth.move(py.path.local(r'c:\tmp\testing3.txt'))
Traceback (most recent call last):
  File "E:\src\python\py\py\_error.py", line 81, in checked_call
    cls = self._geterrnoclass(_winerrnomap[errno])
KeyError: 18

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "E:\src\python\py\py\_path\common.py", line 162, in move
    self.rename(target)
  File "E:\src\python\py\py\_path\local.py", line 443, in rename
    return py.error.checked_call(os.rename, self.strpath, target)
  File "E:\src\python\py\py\_error.py", line 83, in checked_call
    raise value
  File "E:\src\python\py\py\_error.py", line 64, in checked_call
    return func(*args, **kwargs)
OSError: [WinError 17] The system cannot move the file to a different disk drive: 'e:\\tmp\\testing.txt' -> 'c:\\tmp\\testing3.txt'
>>> py.__version__
'1.4.32.dev1'
```

**Comment by smsearcy** [comment]
I think this issue can be closed.  I'm not sure when it was fixed but as of py 1.4.34 (on Python 3.5.2) `py.path.move()` is working on Windows 8 across different drives.

**Comment by RonnyPfannschmidt** [comment]
@smsearcy thanks for the heads up
